Description: Fixes the compatibility with the version of jboss-logging in Debian
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/engine/src/main/java/org/hibernate/validator/internal/util/logging/Log.java
+++ b/engine/src/main/java/org/hibernate/validator/internal/util/logging/Log.java
@@ -33,10 +33,10 @@
 import javax.validation.ValidationException;
 
 import org.jboss.logging.BasicLogger;
-import org.jboss.logging.Cause;
-import org.jboss.logging.LogMessage;
-import org.jboss.logging.Message;
-import org.jboss.logging.MessageLogger;
+import org.jboss.logging.annotations.Cause;
+import org.jboss.logging.annotations.LogMessage;
+import org.jboss.logging.annotations.Message;
+import org.jboss.logging.annotations.MessageLogger;
 
 import static org.jboss.logging.Logger.Level.INFO;
 import static org.jboss.logging.Logger.Level.WARN;
--- a/engine/src/main/java/org/hibernate/validator/internal/util/logging/Messages.java
+++ b/engine/src/main/java/org/hibernate/validator/internal/util/logging/Messages.java
@@ -16,8 +16,8 @@
  */
 package org.hibernate.validator.internal.util.logging;
 
-import org.jboss.logging.Message;
-import org.jboss.logging.MessageBundle;
+import org.jboss.logging.annotations.Message;
+import org.jboss.logging.annotations.MessageBundle;
 
 /**
  * @author Hardy Ferentschik
--- a/engine/pom.xml
+++ b/engine/pom.xml
@@ -49,6 +49,11 @@
             <groupId>org.jboss.logging</groupId>
             <artifactId>jboss-logging</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.jboss.logging</groupId>
+            <artifactId>jboss-logging-annotations</artifactId>
+            <version>debian</version>
+        </dependency>
 
         <!--
         Optional dependencies
